go.mod

```diff
@@ -3,6 +3,7 @@ module github.com/nacos-group/nacos-sdk-go/v2
 go 1.14
 
 require (
+	bou.ke/monkey v1.0.2 // indirect
```

Comment:
Can unit tests be done without introducing monkey

Reply:
Hi @sanxun0325, I notice that `ConnectionEventListener` struct contains a `*NamingGrpcProxy` field and the `NewConnectionEventListener` constructor requires a `NamingGrpcProxy` pointer as input. Can you give me an example for how to mock a `NamingGrpcProxy` instance?

Reply:
Hi @sanxun0325, I notice that `ConnectionEventListener` struct contains a `*NamingGrpcProxy` field and the `NewConnectionEventListener` constructor requires a `NamingGrpcProxy` pointer as input. Can you give me an example for how to mock a `NamingGrpcProxy` instance?

Reply:
@Gerrard-YNWA you can refer to this:https://github.com/nacos-group/nacos-sdk-go/blob/2.0.0/clients/naming_client/naming_grpc/connection_event_listener_test.go

Reply:
Hi @sanxun0325 thanks a lot,  `ConnectionEventListener` use `INamingProxy` interface now, Can I use gomock to write test case?

Reply:
Hi @sanxun0325, I have updated the test case